app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ReturnsController.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
+
+import com.google.inject.Inject
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.views
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+
+class ReturnsController @Inject() (
+    cc: MessagesControllerComponents,
```

Comment:
What is the purpose of this controller?

Reply:
it provides a dummy page which is supposed to represent the start of the returns journey. It's used in a link on the "subscribed" page: https://github.com/hmrc/cgt-property-disposals-frontend/pull/18/files#diff-206ae687ec0b0d08cdda94678976755cR34